OakCityV2/TableViewController.m

```diff
@@ -0,0 +1,216 @@
+                                                            cancelButtonTitle:@"Ok" otherButtonTitles:nil];
+                                         [aV show];
+                                     }];
+    [operation start];
+    
+}
+
+//Alert View
+- (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)buttonIndex {
+    NSString *button = [alertView buttonTitleAtIndex:buttonIndex];
+    
+    if([button isEqualToString:@"Retry"]) {
+        //Fetch JSON
+        NSString *urlAsString = [NSString stringWithFormat:@"https://jobs.github.com/positions.json?description=%@&location=%@", LANGUAGE, TOWN];
+        NSURL *url = [NSURL URLWithString:urlAsString];
+        NSURLRequest *request = [NSURLRequest requestWithURL: url];
+        AFHTTPRequestOperation *operation = [[AFHTTPRequestOperation alloc] initWithRequest:request];
+        operation.responseSerializer = [AFJSONResponseSerializer serializer];
+        
+        //Parse JSON
+        [operation setCompletionBlockWithSuccess:^(AFHTTPRequestOperation *operation, id responseObject) {
+            self.jobs  = [NSMutableArray new];
+            NSArray *responseArray = (NSArray *)responseObject;
+            
+            for (NSDictionary *responseDict in responseArray) {
+                Job* job = [[Job alloc] initWithDictionary:responseDict];
+                [self.jobs addObject:job];
+            }
+            
+            if([self.jobs count] > 0) {
+                [self.tableView reloadData];
+            }
+            
+            //Alert on failure to fetch JSON
+            else {
+                UIAlertView* alert = [[UIAlertView alloc]
+                                      initWithTitle: @"Failed to retrieve data" message: nil delegate: self
+                                      cancelButtonTitle: @"cancel" otherButtonTitles: @"Retry", nil];
+                [alert show];
+            }
+         }
+         
+         //Upon failure
+                                         failure:^(AFHTTPRequestOperation *operation, NSError *error) {
+             UIAlertView *aV = [[UIAlertView alloc]
+                                initWithTitle:@"Error" message:[error localizedDescription] delegate: nil
+                                cancelButtonTitle:@"Ok" otherButtonTitles:nil];
+             [aV show];
+         }];
+        [operation start];
+    }
+    
+}
+
+//Pull to refresh
+- (IBAction)refresh:(UIRefreshControl *)sender {
+    //Fetch JSON
+    NSString *urlAsString = [NSString stringWithFormat:@"https://jobs.github.com/positions.json?description=%@&location=%@", LANGUAGE, TOWN];
+    NSURL *url = [NSURL URLWithString:urlAsString];
+    NSURLRequest *request = [NSURLRequest requestWithURL: url];
+    AFHTTPRequestOperation *operation = [[AFHTTPRequestOperation alloc] initWithRequest:request];
+    operation.responseSerializer = [AFJSONResponseSerializer serializer];
+    
+    //Parse JSON
```

Comment:
Having the same code block, especially a big one like this should be setting off all kinds of alarms.

For your sanity and those who come after you, you really want to follow the DRY principle (https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and don't repeat yourself.  I've seen it in so many place that somebody just cuts and pastes some code because it's a little easier and then the "identical" pieces of code slowly begin to evolve away form each other.